core/src/main/scala/kafka/server/ConfigHelper.scala

```diff
@@ -194,7 +194,8 @@ class ConfigHelper(metadataCache: MetadataCache, config: KafkaConfig, configRepo
     val allSynonyms = {
       val list = configSynonyms(name, allNames, isSensitive)
       if (!groupProps.containsKey(name))
-        list
+        new DescribeConfigsResponseData.DescribeConfigsSynonym().setName(name).setValue(valueAsString)
```

Comment:
Thanks for the review, We need this change because when we do a kafka-configs.sh describe command with --all option, ideally it should display all the configs present in the GroupConfig file and the synonym field should display the source of the config, either it should be `DEFAULT_CONFIG` or `DYNAMIC_GROUP_CONFIG`. But, without this change, it was only displaying the configs, the source for which was `DYNAMIC_GROUP_CONFIG`